content/shared/influxdb3-internals-reference/durability.md

```diff
@@ -0,0 +1,78 @@
+## How Data Flows Through InfluxDB 3
```

Comment:
All headings should be sentence case, following [Google Developer Documentation style](https://developers.google.com/style/headings).